### DISCUSSION TITLE
Validation messages translation

### DISCUSSION THREAD

**Discussion by neflictus** [filamentphp/filament#5759] [Help]
Hi

Admin Panel - 
`return $form->schema([`
`        Grid::make()->columns(4)->schema([`
`                  TextInput::make('year')`
`                      ->rules(['integer', 'min:1920', 'max:2040'])`
`                      ->label(__('Year')),`

How can I change the validation error message for the rules ?

`protected function getMessages(): array`
`    {`
`        return [`
`            'year.min' => __('Year must be greater than 1920'),`
`            'year.max' => __('Year must be smaller than 2040')`
`        ];`
`    }`

Doesn't seem to do the trick.

**Comment by danharrin** [comment]
Prefix the field names with `data.` if the `getMessages()` is on the Page class (LW component).

**Comment by neflictus** [comment]
Thanks. It works.

**Reply by jwhoami** [reply]
Just to add to @danharrin answer, if your rule is in camel case like beforeOrEqual, your field name should be like this

data.field_name.before_or_equal

I tried data.field_name.beforeOrEqual and it does not work
